templates/components/products/reviews.html

```diff
@@ -35,7 +35,7 @@ <h5 itemprop="name" class="productReview-title">{{ title }}</h5>
                             </p>
                         {{/if}}
                     </header>
-                    <p itemprop="reviewBody" class="productReview-body">{{nl2br text}}</p>
+                    <p itemprop="reviewBody" class="productReview-body">{{{ sanitize text }}}</p>
```

Comment:
can we please do the same for the review title as well

Reply:
Sure. I've done the same for the review title.

Reply:
The same issue is also present on facets